com.unity.multiplayer.mlapi/Runtime/Transports/UNET/UnetTransport.cs

```diff
@@ -27,6 +27,17 @@ public enum SendMode
         public int ServerListenPort = 7777;
         public int ServerWebsocketListenPort = 8887;
         public bool SupportWebsocket = false;
+
+        // user-definable channels.  To add your own channel, do something of the form:
```

Comment:
@TwoTenPvP Here's how custom channels can be cleanly and safely specified, building on your idea of having the 'unused' offset